src/views/Search.vue

```diff
@@ -237,13 +260,90 @@ export default {
       }
       return false;
     },
+    filterString() {
```

Comment:
Ideally all the code related to the code generation etc. should be in a separate component, not in the Search component.

Reply:
Actually, even better would be to have model classes for code generation. One method per use case and one class per language. Similarly to what I did here for another project: https://github.com/Open-EO/openeo-web-editor/tree/master/src/export